### PR TITLE
Added Firmware 2021 Accord Hybrid

### PR DESCRIPTION
**Pull request by feyded1020**
***** Template: Fingerprinting for 2021 Honda Accord Hybrid *****

**Description** No support for 2021 Honda Accord Hybrid, added firmwares to values.py

**Verification** Followed fingerprinting wiki

**Route**
Route: None yet

-->

### CODE REVIEW

File: selfdrive/car/honda/values.py [hunk 1]
```diff
@@ -1301,4 +1303,4 @@ class CAR:
 }
 
 HONDA_BOSCH = set([CAR.ACCORD, CAR.ACCORDH, CAR.CIVIC_BOSCH, CAR.CIVIC_BOSCH_DIESEL, CAR.CRV_5G, CAR.CRV_HYBRID, CAR.INSIGHT, CAR.ACURA_RDX_3G])
-HONDA_BOSCH_ALT_BRAKE_SIGNAL = set([CAR.ACCORD, CAR.CRV_5G, CAR.ACURA_RDX_3G])
+HONDA_BOSCH_ALT_BRAKE_SIGNAL = set([CAR.ACCORD, CAR.CRV_5G, CAR.ACURA_RDX_3G])
```
Comment (belm0) [inline selfdrive/car/honda/values.py:1306]: suggest restoring the file's terminating newline-- there shouldn't be a change here

**Comment by adeebshihadeh** [comment]
What's your dongle ID?

File: selfdrive/car/honda/values.py (hunk 1 above)
Comment (adeebshihadeh) [reply selfdrive/car/honda/values.py:1306]: yeah, let's revert this

**Comment by feyded1020** [comment]
> What's your dongle ID?

It is:
1949e3587c4885e7

File: selfdrive/car/honda/values.py (hunk 1 above)
Comment (feyded1020) [reply selfdrive/car/honda/values.py:1306]: Weird, not sure how that happened. I'm still relatively new to this, but all I did was input 2 new lines in Textmate and save it. 

Should I retry and reupload to see if it works correctly?

**Comment by adeebshihadeh** [comment]
Does it work after adding the fingerprint? There's a car port PR open for the 2021 Accords. #20550

**Comment by feyded1020** [comment]
> Does it work after adding the fingerprint? There's a car port PR open for the 2021 Accords. #20550

Yes, car no longer says 'No Exact Match Found', and haven't had any issues with the driving capabilities.

**Comment by csouers** [comment]
@feyded1020 Can you please provide the VIN for the vehicle and also a cabana link that is from the "Copy Share Link" button? You can message me on the comma Discord with the username `@csouers` with any questions.  I am doing the the port for the 2021+ Accord and Odyssey and this is very curious. I would expect to see different radar and camera firmware IDs for this model year.

**Pull request closed without merging**